widgetry/src/event.rs

```diff
@@ -51,24 +51,45 @@ impl Event {
                 position.to_logical(scale_factor).into(),
             )),
             WindowEvent::MouseWheel { delta, .. } => match delta {
+                // "In the beginning" a spinnable mouse wheel was the only input hardware for
```

Comment:
This comment is an opus. I can remove/shorten it, but I thought it might be useful context. I certainly didn't know about these things before digging into this.

Reply:
It's immensely helpful! I wonder if it'd also be a good addition to https://docs.rs/winit/0.22.2/winit/event/enum.MouseScrollDelta.html?